docs/network/evm-nodes.md

```diff
@@ -17,6 +17,37 @@ You can get the kernel by importing it from a running Etherlink Smart Rollup nod
 
 Running an Etherlink EVM node on Etherlink Mainnet requires a computer with 500GB of disk space and at least 16GB RAM.
 
+## Quickstart
+
+If you don't want to see all of the options and settings for the EVM node, you can run the node quickly with these steps:
+
+1. Download the latest release of the `octez-evm-node` binary from from https://gitlab.com/tezos/tezos/-/releases.
+
+1. Run this command to start the node:
+
+   ```bash
+   octez-evm-node run observer \
+     --network <NETWORK> \
+     --history rolling:1 \
+     --data-dir <EVM_DATA_DIR> \
```

Comment:
Can we also drop the `data-dir` and rely on the default option?

Reply:
We recommend 500GB so you should think about where to put it instead of just dumping it somewhere. But it's a quickstart so maybe the default is fine.

Reply:
The default data-dir is not modified by the `--network` selected. I fear not having it explicitly can be confusing because users may expect to be able to start a node for each network. No strong opinion though. If we omit, I would suggest adding a clear warning about where the default data-dir is and the fact that it can only be used for one network at a time.

Reply:
Added this. Do we explicitly need to say to not use the same data dir for mainnet and testnet? It'll fail pretty quickly if you use a dir for one to start a node for the other.

> By default, the node stores its data and configuration file in the folder `$HOME/.octez-evm-node`. You can change these default locations by passing the `--data-dir`, and `--config-file` arguments or setting the equivalent environment variables as described in [Configuring the node](#configuring-the-node).

Reply:
The paragraph LGTM.